_maintenance_update_graph_data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+outFile="$(dirname $0)/js/cert-timeline.tsv"
+
+# Download to a temp file, then move over-top the original, so that the update is
+# atomic from the perspective of the webserver
+curl --silent https://ct.tacticalsecret.com/cert-timeline.tsv >"${outFile}.bak"
+mv "${outFile}.bak" "${outFile}"
```

Comment:
This mv should only happen if the curl is successful, so `curl ... && mv ...`. Alternately you can `set -o errexit` at the top of the script. I don't have strong opinions about which one.